packages/web/integrations/kado/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { FunctionComponent } from "react";
+import { WalletStatus } from "@keplr-wallet/stores";
+import { useStore } from "../../stores";
+import { ModalBaseProps } from "../../modals";
+
+/** Assumed wallet connected */
+export const Kado: FunctionComponent<
+  { assetKey: string } & Pick<ModalBaseProps, "isOpen" | "onRequestClose">
+> = ({ assetKey }) => {
+  const { chainStore, accountStore } = useStore();
+
+  const account = accountStore.getAccount(chainStore.osmosis.chainId);
+
+  if (!(account.walletStatus === WalletStatus.Loaded)) return null;
+
+  return (
+    <iframe
+      src={`https://app.kado.money/?onPayCurrency=USD&onPayAmount=200&onRevCurrency=${assetKey}&offPayCurrency=${assetKey}&offRevCurrency=USD&network=OSMOSIS&onToAddress=${account.bech32Address}&offFromAddress=X&cryptoList=${assetKey}&networkList=OSMOSIS&apiKey=67a2aaad-a2a5-4412-be40-e3a70aa7b53d&product=BUY`}
```

Comment:
1. Is it okay to show api key publicly?
2. I've heard that there are some iframe security concerns. how's your thinking about using iframe?

Reply:
1. I've confirmed with Kado team that it is just a public identifier
2. Any specifics about the concern? This is the recommended method of integration by Kado

Reply:
2. I saw [this article](https://blogs.halodoc.io/iframe-security-threats-and-the-prevention/) but I think Kado is a trusted third party. Ignore it.